src/Domain.jl

```diff
@@ -313,7 +313,7 @@ function ParametrizedTPDomain(;phase::Z,initialconds::Dict{X,Any},constantspecie
         jacobian=zeros(typeof(V),length(phase.species)+1,length(phase.species)+1)
```

Comment:
I'm not able to comment at the exact position due to github limitation.. but species index is also used in line 300 and line 608 (`y0[phase.species[1].index:phase.species[end].index] = ns`), maybe change them as well for consistency.

Reply:
fixed